frida_tools/tracer.py

```diff
@@ -772,16 +775,22 @@ def objc_arg(m):
                 except Exception as e:
                     pass
 
+            if decorate:
+                module_string = ' [%s]' %  function.module.name
+            else:
+                module_string = ''
```

Comment:
Would be good to use double-quoted strings here for consistency. (The weird convention we're following is that strings are double-quoted, and "enum values" are single-quoted.)

Reply:
A good consistent standard -- done!